charts/kafka-connect/values.yaml

```diff
@@ -1,13 +1,13 @@
 ## The image is built with the Dockerfile in /images/kafka-connect
 image:
   repository: us.gcr.io/galoy-org/kafka-connect
-  digest: "sha256:536348d31fdc138012a93b5ad54761c45075952e7c8d31508e629215e000de18"
+  digest: sha256:536348d31fdc138012a93b5ad54761c45075952e7c8d31508e629215e000de18
 
-## The pods from this namespace are allowed to access the Kafka Connect API
-smoketestNamespace: "galoy-dev-smoketest"
+# Change to allow multiple kafka-connect instances
+metadataName: kafka
```

Comment:
What does metadata name mean? Shouldn't it be 'kafkaClusterName' or something?

Reply:
![image](https://github.com/GaloyMoney/charts/assets/43343391/37decc25-561a-421a-91e9-0e1675c3e026)

How about `resourceName` ? Or any better?

Reply:
ok changed to `kafkaConnectInstanceName`

Reply:
changed also the `smoketstNamespace` to `allowedNamespace`